src/main/java/Deadline.java

```diff
@@ -0,0 +1,38 @@
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+
+    private String deadline;
+    private LocalDateTime deadlineDateFormat;
+    public Deadline(String description) throws DukeException {
+        this.fullTaskDescription = description;
+        try {
+            String[] command = description.split(" /by ");
+            if (command.length <= 1) {
+                throw new DukeException("____________________________________________________________\n" +
+                        " OOPS! Your Only Friend cannot take in a deadline entry with no time :(\n" +
+                        "____________________________________________________________\n");
```

Comment:
Line breaks should be before an  operator
```suggestion
                        throw new DukeException("____________________________________________________________\n" 
                                                                   + " OOPS! Your Only Friend cannot take in a deadline entry with no time :(\n" 
                                                                   + "____________________________________________________________\n");
```